recipe/meta.yaml

```diff
@@ -72,3 +73,4 @@ extra:
     - jhamman
     - david-ian-brown
     - khallock
+    - fulminemizzega
```

Comment:
Nice!

Reply:
We should rerender for this to take effect!

Reply:
My bad... this thing is too much automated